### ISSUE TITLE
zoom_sensitivity_ratio and fov_desired

### ISSUE THREAD

**Issue by Perkzitos** [Facepunch/garrysmod-requests#1595]
I have two requests.

1) **zoom_sensitivity_ratio**: It is not saved in the config.cfg file, so my request here is to make GMOD save this command to the config.cfg file. I personally use 1.2.

2) **fov_desired**: make the default value 100, because, why not?

**Comment by viral32111** [comment]
I agree with saving zoom_sensitivity_ratio, I can't see any downside to that convar being saved. But not on changing the **default** FOV value. That would probably lead to alot of confusion and negative feedback from players who don't touch that option. Just leave it at what it's always been and let players choose their own FOV. If they want 100, they select 100. If they want 75, they select 75, etc.

**Comment by robotboy655** [comment]
Made `zoom_sensitivity_ratio` save.

You already opened a duplicate issue for `mat_monitorgamma` and got your answer there.

No to `fov_desired`, there's no reason to touch its default value.
